website/i18n/fr/docusaurus-plugin-content-docs/current/concepts/combining_providers.mdx

```diff
@@ -6,10 +6,10 @@ Avant de commencer, assurez-vous de lire la section sur [les providers](/docs/co
 
 ### Combiner des providers
 
-Nous avons vu précédemment comment créer un simple provider. Mais la réalité est la suivante, 
+Nous avons vu précédemment comment créer un simple provider. Mais la réalité est la suivante,
 dans de nombreuses situations, un provider pourra vouloir lire l'état d'un autre provider.
 
-Pour ce faire, nous pouvons utiliser l'objet [ref] passé au 
```

Comment:
You should keep that one space at the end of a line. I just break those lines (meaning, they're intentional) 'for myselfe' so I can read properly when I'm translating to french. But wait for @rrousselGit 's confirmation.

Reply:
I am really sorry, going to fix it now!